server/core/region.go

```diff
@@ -407,24 +400,18 @@ func (rm *regionMap) Get(id uint64) *RegionInfo {
 	if rm == nil {
 		return nil
 	}
-	if entry, ok := rm.m[id]; ok {
-		return entry.RegionInfo
+	if r, ok := rm.m[id]; ok {
+		return r
 	}
 	return nil
 }
 
 func (rm *regionMap) Put(region *RegionInfo) {
 	if old, ok := rm.m[region.GetID()]; ok {
-		rm.totalSize += region.approximateSize - old.approximateSize
```

Comment:
why to remove `region.approximateSize`

Reply:
It's the same. I also remove the early return.